src/__tests__/resolver.spec.ts

```diff
@@ -34,7 +39,7 @@ export class HttpReader implements Types.IReader {
 }
 
 const runFixtures = (factory: any) => {
-  const dir = `${__dirname}/fixtures/schemas`;
+  const dir = `${__dirname}/fixtures/schemas`.replace(/\\/g, '/');
```

Comment:
`path.join`?
```suggestion
  const dir = path.join(__dirname, '/fixtures/schemas')'
```

Reply:
`__dirname` uses backslashes on Windows. That's why we need the `.replace()`